evidence/rossbulat/0003-rossbulat-retain.md

```diff
@@ -0,0 +1,107 @@
+# Evidence-0003: Retention at Rank 1
+
+|                 |                                                                                             |
+| --------------- | ------------------------------------------------------------------------------------------- |
+| **Report Date** | Date of submission (2024/12/18)                                                             |
+| **Submitted by**| Ross Bulat                                                                        |
+
+
+## Member details
+
+- Matrix username: @rossbulat:matrix.org
+- Polkadot address: 1hYiMW8KSfUYChzCQSPGXvMSyKVqmyvMXqohjKr3oU5PCXF
+- Current rank: 1
+- Date of initial induction: 6th December 2023
+- Date of last report: 21st September 2024
+- Area(s) of Expertise/Interest: Pallets, staking, nomination pools, developer tools, pro-sumer applications.
+
+
+## Reporting period
+
+- Start date: 2024/10/15
+- End date: 2025/01/15
+
+
+## Evidence
+This period has been spent on maintaining the [Staking Dashboard](http://staking.polkadot.cloud) and surrounding tooling ensuring users can still stake on Polkadot.
+
+In this period I have been studying abstraction of code and how to better formalise / organise packages for better maintainability. A range of refactors and upgrades have taken place to keep the codebase using latest packages, and simplifying syntax.
+
+I have focused on user support and engagement, something that has been lacking in the past. Discord has been set up, and over 100 Polkadot users have joined and are communicating staking improvements and requests.
+
+I have  bootstrapped an API service for Polkadot staking, that solves usability issues around staking on Polkadot. User rewards are now fetched instantly. This service will expand to validator performance, fast unstake checking, and validator / pool performance data, that will dramatically speed up workflows. This is a significant improvement over the current state of affairs, where node-only data fetching is becoming unviable as the staking system scales - e.g. 297 to 500 validators, now 30k+ nominators, etc, and where Subscan has strict throttle limits for free API usage that is becoming unviable for high user open source projects like staking dashboard.
+
+Staking dashboard also crossed the milestone of being light client by default. This was necessary to not exhaust RPC connections with the latest JSON RPC spec.
+
+Also in this period, I replaced PJS API with PAPI in the staking dashboard, and rolled out Wallet Connect support. 
+
+More validators have been onboarded into the validator community data source.
+
+PRs Staking Dashboard:
```

Comment:
Generally, it is recommended to only include the most relevant PRs in the argument, instead of providing a full list.

Reply:
Thank you, amended.

Reply:
You should integrate this robotic list of PRs into one or more paragraphs explaining WHAT was done high-level and WHY. Right now it's a dump of links that fellows would have to painstakingly go through and figure out if and why they're important or relevant or not. What is the impact of this long list of PRs?

Reply:
I have summarised the action items in the paragraphs above the list. I would not say the list is painstaking, I have only included the most relevant PRs and the titles are rather self explanatory.
 
I have also touched on the reasoning about this direction in my conclusion. I am happy to expand on ambiguities fellows may have though.

Reply:
looking at what the manifesto is outlining as relevant work, I believe this work is actually valuable for the tooling collective. My current rank does not allow me to vote but I wanted to raise this point for discussion

Reply:
I actually do pallet side work too, which is part of the manifesto. The work outlined in this one evidence was not so much focused on the pallet side development itself but rather making what has been updated system side accessible to Polkadot users. I believe it would be valuable if more fellowship members did this, more of a full stack approach to tackling problems.

I also have no interest in joining the tooling collective at this time.

Reply:
> I actually do pallet side work too, which is part of the manifesto. 

Then this is what should be part of this evidence IMO

>  I believe it would be valuable if more fellowship members did this, more of a full stack approach to tackling the problems we face.

I haven't questioned whether the work is generally useful or not. I'm questioning whether it's relevant for the technical fellowship, which I believe is not (curious what others think). The staking dashboard is one of many useful UI that users can stake on, while w3ux seems to be a library that falls within the umbrella of the tooling collective

Reply:
> Then this is what should be part of this evidence IMO

I cannot control what takes precedence as a priority. If we roll out a breaking change on chain then there should be a focus on making that change accessible. Likewise, if there are major upgrades or modernisation that needs to happen application side, so users can continue securing Polkadot, then that takes precedence.

> I haven't questioned whether the work is generally useful or not. I'm questioning whether it's relevant for the technical fellowship, which I believe is not (curious what others think). The staking dashboard is one of many useful UI that users can stake on, while w3ux seems to be a library that falls within the umbrella of the tooling collective

It has been relevant for the past 9 months, and I don't doubt it is still relevant today. I do pallet development and develop tooling to secure the network in staking. 

I would like to point out too that my development efforts have been completely unfunded since July. I do not have the luxury of spreading my efforts too thin, especially when there are a large amount of critical updates to do. My track record has proven my competence in both polkadot sdk development and application side development, that continues to bring value to the ecosystem.